unittests/files/opt/seagate/sspl/setup/resource_map/lr2/test_storage.py

```diff
@@ -3,22 +3,22 @@
 
 from unittest.mock import patch
 
-from files.opt.seagate.sspl.setup.resource_map.storage import StorageMap
+from files.opt.seagate.sspl.setup.resource_map.lr2.storage import StorageMap
 
 from encl_api_response import (
     ENCLOSURE_RESPONSE, ENCLOSURE_SENSORS_RESPONSE, ENCLOSURE_RESPONSE_EMPTY,
-    ENCLOSURE_NW_RESPONSE)
+    ENCLOSURE_NW_RESPONSE, CONTROLLER_RESPONSE, DRIVE_RESPONSE)
 
 
 class TestStorageMap(unittest.TestCase):
     def setUp(self) -> None:
         self.storage_map = StorageMap()
 
     @patch(
-        "files.opt.seagate.sspl.setup.resource_map.storage.StorageMap.get_realstor_encl_data"
+        "files.opt.seagate.sspl.setup.resource_map.lr2.storage.StorageMap.get_realstor_encl_data"
     )
     def test_get_platform_sensors(self, encl_response):
-        encl_response.return_value = json.loads(ENCLOSURE_SENSORS_RESPONSE)
+        encl_response.return_value = json.loads(ENCLOSURE_SENSORS_RESPONSE)['api-response']['sensors']
```

Comment:
Hi, when we are asserting only on `0th` entry of  `json.loads(ENCLOSURE_SENSORS_RESPONSE)['api-response']['sensors']`, why don't we include only 0th entry or few entries in sample data from `encl_api_response` ?

Reply:
Not, we are not asserting just 0th entry.  Sample data covers sensor types, 'Temperature, Voltage and Current' for platform sensors. Thats why @thavanathan added them. @sumedhak27